### ISSUE TITLE
Green background for search buttons with Umami demo profile

### ISSUE THREAD

**Issue by kepol** [quantcdn/drupal#89]
If you use Quant with the Umami demo profile, then you get green search and clear buttons due to changes in the Umami theme for accessibility. These should be adjusted to look better since this is often what is used for testing.

<img width="1080" alt="Screen Shot 2022-12-04 at 11 06 48 PM" src="https://user-images.githubusercontent.com/282024/205572460-0a4da549-6c38-48d2-9b2b-168e8363fff4.png">
